up42/order.py

```diff
@@ -75,6 +82,39 @@ def is_fulfilled(self) -> bool:
         """
         return self.status == "FULFILLED"
 
+    @property
+    def _stac_search_order(self) -> Tuple[Client, ItemSearch]:
+        url = f"{self.auth._endpoint()}/v2/assets/stac"
+        pystac_client_aux = PySTACAuthClient(auth=self.auth).open(url=url)
+        stac_search_parameters = {
+            "max_items": MAX_ITEM,
+            "limit": LIMIT,
+            "filter": {
+                "op": "=",
+                "args": [
+                    {"property": "order_id"},
+                    self.order_id,
+                ],
+            },
+        }
+        pystac_asset_search = pystac_client_aux.search(filter=stac_search_parameters)
+        return (pystac_client_aux, pystac_asset_search)
+
+    def get_assets(self) -> List[Asset]:
+        """
+        Gets the Order assets or results.
+        """
+        if self.is_fulfilled:
+            _, pystac_asset_search = self._stac_search_order
+            assets = {
```

Comment:
consider renaming the variable to `asset_ids`.
Also, is there a possibility we can get duplicate asset_ids? If not, is there any other advantage of using set instead of list?

Reply:
Accepted :)